docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+
+
+from deepchecks.tabular import Dataset
+
+# We explicitly state the categorical features,
+# otherwise they will be automatically inferred,
+# which may not work perfectly and is not recommended.
+
+# The label can be passed as a column name or
+# as a separate pd.Series / pd.DataFrame
+
+# all metadata attributes are optional.
+# Some checks require specific attributes and otherwise will not run.
+
+train_ds = Dataset(train_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+test_ds = Dataset(test_df, label=label,cat_features=categorical_features, \
+                   index_name=index_name, datetime_name=datetime_name)
+
+#%%
+
+# for convenience lets save it in a dictionary so we can reuse them for future Dataset initializations
+columns_metadata = {'cat_features' : categorical_features, 'index_name': index_name,
+                    'label':label, 'datetime_name':datetime_name}
+
+#%%
+# Run the Deepchecks Suite
+# --------------------------
+#
+# Validate your data with the :func:`deepchecks.tabular.suites.train_test_validation` suite.
+# It runs on two datasets, so you can use it to compare any two batches of data (e.g. train data, test data, a new batch of data
+# that recently arrived)
+#
+# Check out the :doc:`when should you use </getting-started/when_should_you_use>`
+# deepchecks guide for some more info about the existing suites and when to use them.
+
+from deepchecks.tabular.suites import train_test_validation
+
+validation_suite = train_test_validation()
+suite_result = validation_suite.run(train_ds, test_ds)
+# Note: the result can be saved as html using suite_result.save_as_html()
+# or exported to json using suite_result.to_json()
+suite_result
+
+#%%
+# We can see that we have a problem in the way we've split our data!
+# We've mixed up data from two years, causing a leakage of future data
+# in the training dataset.
+# Let's fix this.
+# 
+# Fix Data
+# ^^^^^^^^^^
+
+dt_col = data[datetime_name]
+train_df = data[dt_col.dt.year.isin([2017]) & dt_col.dt.month.isin([6,7,8])]
+test_df = data[dt_col.dt.year.isin([2018]) & dt_col.dt.month.isin([6,7,8])]
+
+#%%
+
+from deepchecks.tabular import Dataset
+
+# Create the new Datasets
+train_ds = Dataset(train_df, **columns_metadata)
+test_ds = Dataset(test_df, **columns_metadata)
+
+#%%
+#
+# Re-run Validation Suite
+# ^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+
+suite_result = validation_suite.run(train_ds, test_ds)
+suite_result
+
+#%%
+#
+# Ok, the date leakage doesn't exist anymore!
+#
+# However, we can see that we have a multivariate drift in the current split, 
+# detected by the :doc:`</checks_gallery/tabular/train_test_validation/plot_whole_dataset_drift>` check.
+# The drift is detected mainly by a combination of features representing the loan's interest rate (``int_rate``) 
+# and its grade (``sub_grade``).
+#
+# We can consider examining other sampling techniques (e.g. using only data from the same year), 
```

Comment:
I think you should say something about the 2 options (keep the split or not) before jumping to discussing each one, otherwise both paragraphs seem a bit disconnected